Database/Updates/World/04-06-20-2017-alterPostionTable-stored.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `ace_world`.`ace_position`
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
```

Comment:
For performance reasons this line should remain VIRTUAL instead of STORED please.

It could also be changed back to a smallint since the expression used here doesn't complain about int. up to you on that one.